packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -111,4 +111,11 @@ Unlike normal disabled buttons, aria-disabled buttons can support tooltips.
 Aria-disabled buttons can operate as links, which also support tooltips.
 
 ```ts file="./ButtonAriaDisabledLinkTooltip.tsx"
+```
+
+### Buttons with a count label
```

Comment:
We could probably omit the "label" text and instead have the example title be "Button with count"